src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/NativeScrollContentPresenter.iOS.cs

```diff
@@ -156,11 +156,332 @@ public override void SetNeedsLayout()
 			}
 		}
 
-		
+		#region Layouting
```

Comment:
this should be a rename rather than a new file, to preserve the source history.

Reply:
Actually I just merged back the NSCP.iOS.cs and NSCP.iOSmacOS.cs into the NSCP.iOS.cs (basically it's undoing [this commit](https://github.com/unoplatform/uno/commit/f8f1af4f422ff16bac0913d14b38aa5077ee37b4#diff-3de33fe97a73c0cb0a1737c9be7866a002b5a81266402df54af175e296cdc561)). The only solution for what you are saying would be to rename it like NSCP.layouting.iOS.cs ... not sure we have any interest to keep that separated.

Reply:
ok, never mind then